client/me/concierge/calendar-card.js

```diff
@@ -62,7 +65,7 @@ class CalendarCard extends Component {
 
 	renderHeader = () => {
 		// The "Header" is that part of the foldable card that you click on to expand it.
-		const date = moment( this.props.date );
+		const date = moment( this.props.date ).tz( this.props.signupForm.timezone );
```

Comment:
Not a huge deal but maybe we can extract this out into a method since its repeated 3 times in this file.

maybe:
```
withTimezone = dateTime => moment( dateTime ).tz( this.props.signupForm.timezone );
```

Then we can just use 
`const date = this.withTimezone( this.props.date );`
`const today = this.withTimezone().startOf( 'day' )` and
`{ this.withTimezone( time ).format( 'h:mma z' ) }`

Reply:
Great suggestion, updated.